semgrep-core/src/osemgrep/cli_scan/Output.ml

```diff
@@ -21,6 +35,25 @@ let output_result (conf : Scan_CLI.conf) (res : Core_runner.result) : unit =
   let cli_output : Out.cli_output =
     Cli_json_output.cli_output_of_core_results conf res
   in
+  (if conf.autofix then
+   (* TODO report when we fail to apply a fix because it overlaps with another?
+    * Currently it looks like the Python CLI will just blindly apply
+    * overlapping fixes, probably breaking code.
+    *
+    * At some point we could re-run Semgrep on all files where some fixes
+    * haven't been applied because they overlap with other fixes, and repeat
+    * until all are applied (with some bounds to prevent divergence). This
+    * probably happens rarely enough that it would be very fast, and it would
+    * make the autofix experience better.
+    * *)
+   let modified_files, _failed_fixes = apply_fixes conf cli_output in
+   if not conf.dryrun then
+     if modified_files <> [] then
+       pr2
```

Comment:
You should now be able to use the Logs library. See my recent PR regarding logging.

Reply:
I tried to faithfully reproduce the Python behavior here. It prints on stderr with no additional text other than what's seen here. I just tried out the Logs library and got:

```
osemgrep: [WARNING] successfully modified 1 file.
```

I had to use `Logs.warn` because otherwise it looks like this wouldn't get printed by default. This output seems weird and it's inconsistent with the existing output.

I guess I'm not sure how faithfully we are trying to reproduce the current behavior. Does it matter whether this is slightly different? If I were writing this from scratch I would probably use `Logs.info` for this, so it appears only when `--verbose` or higher is passed.

Anyway, I'm going to merge this without using Logs here but I'll be happy to make a followup PR depending on your answer here.